x/wasm/keeper/testdata/version.txt

```diff
@@ -1,3 +1,2 @@
-v1.3.0
+v1.4.0-rc.1
 burner.wasm: v1.2.0
```

Comment:
You can update burner.wasm too now (https://github.com/CosmWasm/cosmwasm/issues/1783#issuecomment-1697302795)

Reply:
I have tried with burner v1.4.0 but not all data is erased. https://github.com/CosmWasm/wasmd/blob/v0.41.0/x/wasm/keeper/keeper_test.go#L1418-L1419 fails

Reply:
Did you apply the limit as shown in the linked diff? Maybe it needs to be higher. You can put 100 or 1000 there.

Yeah the "action" attribute has changed to "migrate".